src/current/_includes/releases/v25.1/v25.1.0-alpha.3.md

```diff
@@ -0,0 +1,76 @@
+## v25.1.0-alpha.3
+
+Release Date: January 15, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v25-1-0-alpha-3-backward-incompatible-changes">Backward-incompatible changes</h3>
+
+- Several metrics are redundant and have been removed. Below is a list mapping from each removed metric to an existing, identical metric. [#138786][#138786]
+  - Removed `sql.schema_changer.running`, which is redundant with `jobs.schema_change.currently_running`.
+  - Removed `sql.schema_changer.successes`, which is redundant with `jobs.schema_change.resume_completed`.
+  - Removed `sql.schema_changer.retry_errors`, which is redundant with `jobs.schema_change.resume_retry_error`.
+  - Removed `sql.schema_changer.permanent_errors`, which is redundant with `jobs.schema_change.resume_failed`.
+
+<h3 id="v25-1-0-alpha-3-general-changes">General changes</h3>
+
+- When changefeeds are created with a resolved option lower than the `min_checkpoint_frequency` option, a notice is printed to inform the user since this may lead to unexpected behavior. [#138181][#138181]
```

Comment:
```suggestion
- When changefeeds are created with a resolved option lower than the `min_checkpoint_frequency` option, a notice is now printed, since this may lead to unexpected behavior. [#138181][#138181]
```